MariiaNovitska/HW5_loops/task_2.py

```diff
@@ -0,0 +1,17 @@
+input_number = int(input("How many Fibonacci numbers do you want to see? "))
```

Comment:
good job
look at this code:


input_number = int(input("How many Fibonacci numbers do you want to see? "))

if input_number == 0:
    print("Sooo, you don't want to see Fibonacci numbers? OK, it's your choice)")
elif input_number == 1:
    print([0])
else:
    num = [0, 1]
    for _ in range(input_number - 2):
        num.append(num[-1] + num[-2])
    print(num)

Reply:
understood